posts/models.py

```diff
@@ -502,16 +502,21 @@ def set_actual_close_time(self):
             else:
                 self.actual_close_time = max(close_times)
         elif self.conditional:
-            self.actual_close_time = min(
-                filter(
-                    bool,
-                    [
-                        self.conditional.condition.actual_close_time,
-                        self.conditional.condition_child.actual_close_time,
-                    ],
-                ),
-                default=None,
-            )
+            if self.conditional.condition.resolution:
+                self.actual_close_time = min(
+                    filter(
+                        bool,
+                        [
+                            self.conditional.condition.actual_close_time,
+                            self.conditional.condition_child.actual_close_time,
+                        ],
+                    ),
+                    default=None,
+                )
+            else:
+                self.actual_close_time = (
+                    self.conditional.condition_child.actual_close_time
+                )
```

Comment:
I think this might be incorrect for cases when Condition Parent was closed before child and has not been resolved yet. 

We want to close the entire Conditional Question if any of Parent/Child was closed/resolved.
Related ticket -- https://github.com/Metaculus/metaculus/issues/1008